app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Admin::CategoriesController < Admin::BaseController
+  before_action :get_categories, only: %i(index destroy)
```

Comment:
```suggestion
  before_action :get_categories, only: :index
```
get_categories ở destroy làm gì nhỉ b?

Reply:
Mình làm destroy ajax bạn ạ.